dashboard/app/views/layouts/_header.html.haml

```diff
@@ -85,7 +85,7 @@
             - else
               .headerlinks.hide_on_tablet
                 - Hamburger.get_header_contents(header_contents_options).each do |entry|
-                  %a.headerlink{id: entry[:id], href: entry[:url]}= entry[:title]
+                  %a.headerlink{id: entry[:id], onclick: "event.preventDefault(); logToFirehose({nav_menu_clicked: '#{entry[:title]}'}, '#{entry[:url]}');",  href: entry[:url]}= entry[:title]
 
```

Comment:
I'm unclear as to why we need both the title and url - isn't the url always the same for a given title?  For example, "Professional Learning" always goes to https://code.org/educate/professional-learning.

Reply:
Moreover, I'd expect you to be logging the id rather than the title since the title will be the localized string, ie https://github.com/code-dot-org/code-dot-org/blob/9863bc05ec501b2b7d7a83cd4a7ad8c7ac5c1b6b/lib/cdo/hamburger.rb#L247 so a click on "Professional Development" in English would be counted separately from a click on that link in, say, Spanish.